python/feathub/feature_views/transforms/feature_view_generated_feature_transform.py

```diff
@@ -0,0 +1,31 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Dict
+
+from feathub.feature_views.transforms.transformation import Transformation
+
+
+class _FeatureViewGeneratedFeatureTransform(Transformation):
```

Comment:
Instead of adding a new subclass of `Transform`, would it be simpler to just reuse `ExpressionTransform` and add a built-in-function that generates the timestamp value?

Reply:
I think it makes sense to add a built-in function such as `CURRENT_EVENT_TIME` that generates the timestamp value. 

However, I found that it is not trivial to support it with the current implementation of FlinkAstEvaluator, which naively passes the function call to Flink SQL expression. And the `CURRENT_EVENT_TIME` doesn't exist in Flink SQL expression. We need to refactor FlinkAstEvaluator to properly handle the `CURRENT_EVENT_TIME` function, which requires non-trivial work. And I think it should not block this PR. 

Therefore, the `CURRENT_EVENT_TIME` is only used internally for now. And we can expose it as a built-in function to users after we can handle `CURRENT_EVENT_TIME` properly. What do you think?

Reply:
Sounds good. Thanks for the explanation.